Description: Man pages are not generated by upstream makefile, add .PHONY entry.
Author: Francesco Paolo Lovergine <frankie@debian.org>
Last-Update: 2008-01-08
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -235,3 +235,4 @@ endif # HAVE_LD_SHARED=no
 
 endif # HAVE_LIBTOOL=no 
 
+.PHONY: man
